### ISSUE TITLE
host.registerObserver(TestChangeFeedObserver.class) vs host.registerObserver(new TestChangeFeedObserver());

### ISSUE THREAD

**Issue by ghost** [Azure/azure-documentdb-changefeedprocessor-java#33]
looking over your sample I wonder why are you are doing this:
host.registerObserver(TestChangeFeedObserver.class);
 
instead of
host.registerObserver(new
TestChangeFeedObserver());

**Comment by PraffulDhoke** [comment]
I agree, we can inject spring manged bean in the way. Right now we are facing issue  to inject spring manged beans in the observer.

Thanks,
Prafful D
